### ISSUE TITLE
[BUG] Account changer is broken

### ISSUE THREAD

**Issue by UnzorZ** [John200410/rusherhack-issues#1251]
**Describe the bug**
You cant login into other accounts with the account manager of the main menu

**To Reproduce**
Just try to login

**Other mods**
zero

**Expected behavior**
Login into the account

**Screenshots**
https://imgur.com/undefined

**Additional context**

**Comment by Ozynt** [comment]
Make sure you password and microsoft email is correct and the account doesn't have 2FA and the age on the account is >= 18. 
I have a few questions for you. 
Did the alt manager work before? Are you from Russia?

**Comment by UnzorZ** [comment]
The email and password works. I dont have 2FA and the age is +18. Yes, the alt manager used to work for me and a friend, and none of us can use it anymore. No, im not from Russia

**Comment by UnzorZ** [comment]
also the photo is here: https://imgur.com/a/tKgg6Ht

**Comment by Ozynt** [comment]
Have did you try re-adding them?

**Comment by UnzorZ** [comment]
Yeah, I tried but is the same, it can't log in

**Comment by Mathrandom7910** [comment]
I came on here to report this exact issue, I am getting the same error. MS may have changed some authentication things with migration.

**Comment by Ozynt** [comment]
Yes, seems like a lot of people are having issues. John will probably look into solving this issue soon.

**Comment by Ozynt** [comment]
Also what launcher are you using?

**Comment by Ozynt** [comment]
Microsoft store version or the old one?

**Comment by Mathrandom7910** [comment]
i use the old one (7/8) on windows 11, I could never install the one from the store

**Comment by John200410** [comment]
Microsoft has changed the certificate on their xboxlive auth servers used for Minecraft authentication. A possible solution is being looked into.

**Comment by John200410** [comment]
In the meantime, you can change the JVM that your Minecraft profile uses as a temporary solution

**Comment by Ozynt** [comment]
This has been fixed in the next update.
